app/run.py

```diff
@@ -60,16 +77,16 @@ def calculate_gallons_required(formatted_data):
     :param formatted_data: An integer for the number of feet required to paint
     :return: feet / paint coverage, rounded up
     """
-    return math.floor(formatted_data['ft'] / 350)
+    return math.ceil(formatted_data['ft'] / 400)
```

Comment:
Description says 400ft per gallon and I also googled it to make sure it was correct.
Also, rounding up is better for when the room is less than 400 ft so it returns at least 1 gallon and its better for the user to have extra paint :)